ibis/backends/tests/test_udf.py

```diff
@@ -31,10 +37,16 @@
     reason="Docker image has Python 3.10, results in `cloudpickle` version mismatch",
 )
 
+pyspark_version_mismatch = mark.notimpl(
+    ["pyspark"],
+    condition=PYSPARK_LT_35,
+    raises=NotImplementedError,
+    reason="pyarrow UDFs require PySpark 3.5+",
+)
+
```

Comment:
```suggestion
```

Should be able to git rid of these if you use the `xfail_version` marker as suggested below.